pkg/doc/vc/crypto/crypto.go

```diff
@@ -154,6 +171,16 @@ func (c *Crypto) SignCredential(
 	signerData *vc.Signer, vc *verifiable.Credential, opts ...SigningOpts) (*verifiable.Credential, error) {
 	switch signerData.Format {
 	case vcsverifiable.Jwt:
+		if signerData.SDJWT.Enable {
+			sdJWTDigests, err := c.getSDJWTCredentialSubjectDigests(vc, signerData.SDJWT.HashAlg)
+			if err != nil {
+				return nil, err
+			}
+
+			vc.Subject = sdJWTDigests.Subject
+			opts = append(opts, WithSDJWTDisclosures(sdJWTDigests.Disclosures))
+		}
+
```

Comment:
if sd-jwt is enabled for given profile, I'm creating vc.SUbject with digests and also add `WithSDJWTDisclosures(sdJWTDigests.Disclosures)` as option

Reply:
lets discuss this